.gitignore

```diff
@@ -8,6 +8,7 @@
 /*.yml
 /config
 /junit.xml
+/elastic-detection-cli
```

Comment:
I'm assuming this was just on accident. Since it's local to your setup, I think we can restore this line 
```suggestion
```